canonical-web-and-design/vanilla-framework
# Update Travis and Percy configs to run on Flask docs

Updates travis configuration to build and run docs as flask based app.
Adds a script to get all the links from docs and snapshot them to Percy (using PercyScript).

### QA

- make sure travis build passes
- make sure snapshots of all pages (all docs and examples) are submitted to Percy

NOTE: 
Percy shows 383 missing snapshots agains 381 new snapshots.
Missing snapshots are 404 page (which crawler doesn't find obviously) and old code-snippets page, which in Jekyll site was an empty HTML with a redirect.
